prime-router/metadata/schemas/Experity/CUC-AL-covid-19.schema

```diff
@@ -43,3 +46,17 @@ elements:
           display: "Strep A Ag - rapid"
         - code: 99999999
           display: "UA Dipstick Auto w/o Micro"
+
+    - name: patient_county
```

Comment:
I almost feel this field should be added to a higher level schema.  Either covid 19 or a new Experity schema that gets inherited (I am thinking extends keyword) by the others.

Reply:
My reason for putting this in the Child schema is so that I can control the cardinality by Experity Sender.  Compass is currently in Production and they don't have problems with data entry for the zip code, so we need to let their results pass even if our Zip table is not up-to-date.  Currently our Zip table is out-of-date, so almost all submissions fail even though the zip codes are correct.

Mo has created a ticket to change how the zip code table is maintained and updated.

Reply:
This translation could be moved up to the Experity level, right @rhood23699?  Unless it's the norm that Experity clients use the same orderable for different tests?

Reply:
Experity clients use same orderable for different tests, so this need to remain at the Child level.

Reply:
Got it!